dbt_meshify/dbt_projects.py

```diff
@@ -390,6 +391,23 @@ def split(
 
         return subproject
 
+    @property
+    def group_definition_files(self) -> List[Path]:
+        """A list of files that contain group definitions, in order of how many groups are present."""
+
+        if self._group_definition_files is not None:
+            return self._group_definition_files
+
+        paths: Dict[Path, int] = {}
+        for group in self.manifest.groups.values():
+            path = self.path / Path(group.original_file_path)
+            paths[path] = paths.get(path, 0) + 1
+
+        self._group_definition_files = list(
```

Comment:
Yep. Looking at it now, however, I realize that sorting actually isn't necessary, since it should error out if `len(x) > 1`. I can pull this out to free up a few milliseconds